README.md

```diff
@@ -33,16 +33,12 @@ A snapchain node lets you read and write messages to the network. You will need
 - A public IP address
 - Ports 3381 - 3383 exposed on both TCP and UDP. 
 
-You can start a new node or upgrade an existing node with the following commands: 
+You can start a new node or upgrade an existing node with the following command: 
 
 ```bash
-mkdir snapchain
-cd snapchain
-docker compose down # If you have a previous version running
-wget https://raw.githubusercontent.com/farcasterxyz/snapchain/refs/heads/main/docker-compose.mainnet.yml -O docker-compose.yml
-docker compose up  # append -d to run in the background
+curl -sSL https://raw.githubusercontent.com/farcasterxyz/snapchain/refs/heads/main/scripts/snapchain-bootstrap.sh | bash
```

Comment:
Don't you have to also download the snapchain.sh script? I'm having issues running snapchain-bootstrap.sh because it says snapchain.sh is missing.